src/com_tjnotifications/admin/models/notifications.php

```diff
@@ -85,11 +85,15 @@ protected function populateState($ordering = 'id', $direction = 'asc')
 		parent::populateState($ordering, $direction);
 
 		// Get pagination request variables
-		$limit = $app->getUserStateFromRequest($this->context . '.list.limit', 'limit', $app->get('list_limit'), 'int');
+		$limit = (isset($app->input->get('list')['limit'])) ? $app->input->get('list')['limit'] : $app->getUserStateFromRequest('global.list.limit', 'limit', $app->get('list_limit'), 'int');
```

Comment:
@deepa-g In $app->input->get('list')['limit'] the $app->input->get('list') should have some default value and the filter or else it will give error if list is not defined same for $app->get('list_limit')